autogen/lua_definitions/functions.lua

```diff
@@ -9340,26 +9341,30 @@ end
 --- @param y number
 --- @param z number
 --- @return number
+--- Finds the height of the highest floor below a given position (x, y, z). If no floor is found, returns the default floor height of -11000.
 function find_floor_height(x, y, z)
     -- ...
 end
 
 --- @param x number
 --- @param z number
 --- @return number
+--- Finds the height of the poison gas at a given position (x, z), if the position is within a gas region. If no gas is found, returns the default height of -11000.
 function find_poison_gas_level(x, z)
     -- ...
 end
 
 --- @param colData WallCollisionData
 --- @return integer
+--- Detects wall collisions at a given position and adjusts the position based on the walls found. Returns the number of wall collisions detected.
 function find_wall_collisions(colData)
     -- ...
 end
 
 --- @param x number
 --- @param z number
 --- @return number
+--- Finds the height of water at a given position (x, z), if the position is within a water region. If no water is found, returns the default height of -11000.
```

Comment:
same here, but with "`gLevelValues.floorLowerLimit` (default -11000)"

Reply:
Fixed this